src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/endring-av-navn/config/authorization/policy.xml

```diff
@@ -131,6 +186,12 @@
             <xacml:AttributeDesignator AttributeId="urn:oasis:names:tc:xacml:1.0:action:action-id" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:action" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false"/>
           </xacml:Match>
         </xacml:AllOf>
+        <xacml:AllOf>
+          <xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+            <xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">complete</xacml:AttributeValue>
+            <xacml:AttributeDesignator AttributeId="urn:oasis:names:tc:xacml:1.0:action:action-id" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:action" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false"/>
+          </xacml:Match>
+        </xacml:AllOf>
```

Comment:
Consider adding this to all the test policies.. We tend to copy/pase the policies a lot when creating new test data, so it would be good if this rule was included to.

Reply:
I changed the rule to be more inline with correct use. Copied for the app template as that was correct. And then added the rule to almost all applications.